src/dox/Dox.hx

```diff
@@ -65,6 +65,9 @@ class Dox {
 			@doc("Disable markdown rendering")
 			["--no-markdown"] => function() cfg.useMarkdown = false,
 
+			@doc("Show private fields")
+			["--private"] => function() cfg.showPrivate = true,
```

Comment:
Maybe `--include-private` would be a better name, since we already have `--include`?

Reply:
Sounds good